apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UpdateBucketOptions.cs

```diff
@@ -87,7 +92,6 @@ internal void ModifyRequest(UpdateRequest request, Bucket bucket)
                 if (IfMetagenerationMatch != null)
                 {
                     request.IfMetagenerationMatch = IfMetagenerationMatch;
-                    RetryHandler.MarkAsRetriable(request);
```

Comment:
We have removed the handler but haven't added it back later. Is it deliberate? If yes, please add a comment.